src/shared/Elements/CTAButton.tsx

```diff
@@ -0,0 +1,80 @@
+import styled from '../Styles/styled-components';
+
+export interface IButtonProps {
+  secondary?: boolean;
+  isLoading?: boolean;
+  isNarrow?: boolean;
+  disabled?: boolean;
+}
+
+export const CTAButton = styled.button<IButtonProps>`
+  background-color: ${(props) =>
+    props.secondary ? props.theme.colors.black60 : props.theme.colors.red};
+  font-size: 14px;
+  font-family: ${(props) => props.theme.fonts.header};
+  color: white;
+  padding: 10px 15px;
```

Comment:
nit: can we use increments of 2 for scaling purposes?

Reply:
so 16px instead of 15px?

Reply:
@loreina

Reply:
up to you, play around with it